src/partition/repart.c

```diff
@@ -2058,7 +2067,8 @@ static int context_dump_partitions(Context *context, const char *node) {
                                 TABLE_UINT64, p->current_padding == UINT64_MAX ? 0 : p->current_padding,
                                 TABLE_UINT64, p->new_padding,
                                 TABLE_STRING, padding_change, TABLE_SET_COLOR, !p->partitions_next && sum_padding > 0 ? ansi_underline() : NULL,
-                                TABLE_STRING, activity ?: "unchanged");
+                                TABLE_STRING, activity ?: "unchanged",
+                                TABLE_STRV, p->drop_in_files);
```

Comment:
hmm, what's the point of displaying this? I mean, it's great we read it, but outside of debugging, I doubt this information should show up. Maybe in json output, but certainly not in the regular text output? (if you want to keep this in the json output, then hide it otherwise via table_hide_column_from_display())

Reply:
Does it really hurt to include it in the text output? I figure it's useful for users to be able to see all dropins that apply to some partition config

Reply:
at the very least hide the column of not a single partition has drop-ins. That should suffice for me